### ISSUE TITLE
My GitHub handle shows up in unknown contributors list even after update to affiliations

### ISSUE THREAD

**Issue by PushkarJ** [cncf/gitdm#161]
## Problem
I thought my name would be gone from here: https://github.com/cncf/gitdm/blob/aef91b70172db505bf9e5380df23ee5d69a2182a/src/gha_unknown_contributors_recent.csv#L1085 after this: #143 was merged. But it does not seem like that has happened

### Possible side-effect
I am also not showing up in https://all.devstats.cncf.io/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=First%20CNCF%20project%20join%20date%20-%20now&var-metric=contributions&var-repogroup_name=CNCF&var-country_name=All and https://all.devstats.cncf.io/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=First%20CNCF%20project%20join%20date%20-%20now&var-metric=contributions&var-repogroup_name=Kubernetes&var-country_name=All even though I have made contributions in both the orgs and have seen my name in the list in the past. I thought this was because I was flagged as "unknown" contributor but not sure if it is because of that or a separate issue. 

PS: Thank you for maintaining this humongous database for CNCF. I really appreciate your work!

**Comment by lukaszgryglicki** [comment]
1. We don't update the `gha_unknown_contributors.csv` file, it is just a file from some past processing. All that needs to be updated are text affiliations files as explained in [this workflow](https://github.com/cncf/gitdm#addingupdating-affiliation), other files like `github_users.json` and `cncf-config/email-map` are updated automatically.
2. You will be visible in up to 4 weeks after your affiliation data is merged, we update affiliations data [that often](https://github.com/cncf/gitdm#addingupdating-affiliation):
```
Please note that cncf/gitdm affiliations are imported into DevStats (cncf/devstats) once per 4 weeks.
```

So I'm closing this, please reopen if your data is not visible even after that amount of time, so I will then check manually.

**Comment by PushkarJ** [comment]
Thank you @lukaszgryglicki for the explanation. It's been more than two months since the affiliation PR was merged on August 15 #143. Unfortunately, I still do not see my name in both the links above. Any chance you can re-open this and investigate?

**Comment by PushkarJ** [comment]
Never mind. I see my github handle is back now !! Thank you!
